pinot-common/src/main/java/org/apache/pinot/common/config/SloConfig.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.config;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.lang.reflect.Field;
+import org.apache.pinot.common.utils.EqualityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Section of the table config that provides service-level objectives for specific dimensions such as
+ * latency, freshness.
+ *
+ * Note that analytical query latencies can be highly variable - the SLO should include the maximum tolerable latency
+ * for servicing the common, high latency queries.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SloConfig {
+  private static final Logger LOGGER = LoggerFactory.getLogger(SloConfig.class);
+
+  @ConfigKey("latencyMs")
+  @ConfigDoc("Latency SLO in milliseconds for queries of this table")
+  private long latencyMs;
+
+  @ConfigKey("freshnessLagMs")
+  @ConfigDoc("Freshness Lag SLO in milliseconds for queries of this table (realtime only)")
+  private long freshnessLagMs;
```

Comment:
Consider allowing the user to specify freshness lag in terms of a string such as : "15m" or "1h". See TimeUtils.java

Reply:
Will do